choicepy/__init__.py

```diff
@@ -1,593 +1,616 @@
+                    if can1 == can2:
+                        pairwise_comp_matrix[can1, can2] = np.nan
+                    else:
+                        comparision = int(voter.index(self.candidates[can1]) < voter.index(self.candidates[can2]))
+                        pairwise_comp_matrix[can1, can2] += comparision
+        
+        pairwise_positive_matrix = pairwise_comp_matrix - self.num_voters/2 
+
+        winners = []
+
+        for c, candidate in enumerate(self.candidates):
+            
+            if np.all(np.delete(pairwise_positive_matrix[c,:],c) > 0):
+                winners.append(candidate)
+        #print(pairwise_comp_matrix)
+        return winners
+
+    def generate_borda_rule(self, point_distribution):
+        """
+        This function gets a string and returns the corresponding list of points used in the Borda-count rule.
+        :param point_distribution: string, valid inputs: "borda_0", "borda_1", "dowdall"
+        :return: a list
+            -"borda_0": returns [n-1,n-2,...,0]
+            -"borda_1": returns [n,n-1,...,1]
+            -"dowdall": returns [1, 1/2, 1/3, ..., 1/n]
+            (n = len(self.candidates))
+        """
+        if point_distribution == "borda_0":
+            points_list = [len(self.candidates) - i for i in range(1, len(self.candidates) + 1)]
+        elif point_distribution == "borda_1":
+            points_list = [len(self.candidates) - i for i in range(0, len(self.candidates))]
+        elif point_distribution == "dowdall":
+            points_list = [1 / i for i in range(1, len(self.candidates) + 1)]
+        return points_list
+
+    def borda(self, points_list = None):
+        """
+        Elects a winner according to the weighted rankings of the voters.
+        :param points_list: a list (for example the output of generate_borda_rule(self, point_distribution))
+        :return: a string (the name of the winner of the election)
+        """
+            
+        voters = np.array(self.voters)
+        if points_list is None:
+            points_list = self.generate_borda_rule("borda_0")
+
+        cumulated = Counter(voters[:, 0])
+        for key in cumulated:
+            cumulated[key] *= points_list[0]
+        for v in range(1, len(voters[0][:])):
+            count = Counter(voters[:, v])
+            for key in count:
+                count[key] *= points_list[v]
+            cumulated += count
+        max_value = max(cumulated.values())
+        return sorted(key for key, value in cumulated.items() if value == max_value)
+
+    def elect(self, rule):
+        if rule == "dictator":
+            return self.dictator()
+
+        if rule == "condorcet":
+            return self.condorcet()
+
+        if rule == "approval":
+            return self.approval()
+
+        if rule == "plurality":
+            return self.plurality()
+
+        if rule == "borda":
+            return self.borda()
+
+
+my_profile = Profile()
```

Comment:
Could you please move this part to `./tests/`. It would be useful if  you create a flie called `manual_tests` which starts like this:

```
import choicepy
my_profile = choicepy.Profile()
```

Then the rest should work. And if the working directory is set to the git repo, then the import should work fine.